### ISSUE TITLE
History of spinal cord injury (SCI)

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10804]
**Preferred term label:**
History of spinal cord injury (SCI)

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Deep vein thrombosis (DVT)
Heterotopic ossification (HO), nongenetic
Pneumonia, bacterial
UTI: Urinary tract infection, symptomatic
Depression, endogenous

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6001135
- label: History of spinal cord injury
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: PMID:32809556
- def: A medical history of hving had a spinal cord injury prior to the presenting complaint.
- comment: Spinal cord injury (SCI) results from direct or indirect spinal cord damage. The most common SCI cause is acute trauma from motor vehicular crashes (MVCs), although the condition may also arise from insidious etiologies such as malignancies and chronic tuberculous infection. Spinal cord lesions may lead to permanent disability, significant morbidity, and even mortality. High spinal injuries often impair cardiorespiratory function, requiring emergent interventions.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10804
- synonyms:
